docs/about/faq.md

```diff
@@ -128,7 +136,7 @@ have a technical solution yet. Possibilities are to be checked by us in the near
 
 
 
-## Equipment Auditorium Market Place
+## Auditorium for keynotes and market place. equipment: 
```

Comment:
```suggestion
## Auditorium equipment (keynotes and market place)
```

Reply:
What's wrong with my suggestion?

Reply:
Nothing, I didn't see your suggestion and wanted to correct Dirk's suggestion. Meanwhile I reviewed and approved

Reply:
Normally, the initial contributor would correct any issues his/herself.

As a reviewer, I prefer to only make suggestions, in order to avoid situations like these...